aim/api/resource.py

```diff
@@ -256,3 +256,113 @@ def __init__(self, **kwargs):
                                              'provided_contract_names': [],
                                              'consumed_contract_names': []},
                                             **kwargs)
+
+
+class Filter(AciResourceBase):
+    """Resource representing a contract filter in ACI.
+
+    Identity attributes: name of ACI tenant and name of filter.
+    """
+
+    identity_attributes = ['tenant_name', 'name']
+    other_attributes = ['display_name']
+
+    _aci_mo_name = 'vzFilter'
+    _tree_parent = Tenant
+
+    def __init__(self, **kwargs):
+        super(Filter, self).__init__({}, **kwargs)
+
+
+class FilterEntry(AciResourceBase):
+    """Resource representing a classifier entry of a filter in ACI.
+
+    Identity attributes: name of ACI tenant, name of filter and name of entry.
+
+    Values for classification fields may be integers as per standards
+    (e.g. ip_protocol = 6 for TCP, 17 for UDP), or special strings listed
+    below. UNSPECIFIED may be used to indicate that a particular
```

Comment:
That's right - an entry like FilterEntry(tenant_name=..., filter_name=..., name=...) is an allow-all entry.